notebooks/official/experiments/get_started_with_comparing_local_trained_models.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<p>I am not sure what you mean by "time series". This does not appear to be a forecasting model. Can you clarify?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>"Time series" is way to represent the logging of "the loss and metrics on every epoch to TensorBoard". I changed accordingly. </p>